Dockerfile

```
@@ -2,18 +2,18 @@ FROM node:20-alpine
 
 ENV EXTERNAL_PORT 3000
 ENV PORT 3000
-ENV NODE_ENV production
 
 WORKDIR /app
 
 HEALTHCHECK --interval=30s --timeout=3s \
   CMD wget --spider http://localhost:${PORT}/_healthz || exit 1
```

My healthchecks were failing with this call b/c it was falling back to IPv6 (as seen in `docker inspect --format "{{json .State.Health }}" cht-user-management-cht-user-management-1 |jq`):

```
Connecting to localhost:3000 ([::1]:3000)
wget: can't connect to remote host: Connection refused\n
```

We can force it to use IPv4 and the healthcheck passes:

```suggestion
HEALTHCHECK --interval=30s --timeout=3s \
  CMD wget --spider http://127.0.0.1:${PORT}/_healthz || exit 1
```